packages/react-native-web/src/exports/Modal/ModalFocusTrap.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (c) Nicolas Gallagher.
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import React, { useRef, useCallback, useEffect } from 'react';
+
+import { canUseDOM } from 'fbjs/lib/ExecutionEnvironment';
+
+import View from '../View';
+import createElement from '../createElement';
+import StyleSheet from '../StyleSheet';
+
+/**
+ * This Component is used to "wrap" the modal we're opening
+ * so that changing focus via tab will never leave the document.
+ *
+ * This allows us to properly trap the focus within a modal
+ * even if the modal is at the start or end of a document.
+ */
+
+const FocusBracket = () => {
+  return createElement(
+    'div',
+    {
+      style: styles.focusBracket,
+
+      // Sets `aria-hidden` to true & remove the semantic meaning
+      accessible: true,
+      importantForAccessibility: 'no-hide-descendants',
```

Comment:
There doesn't seem to be much point using these props if you're setting tabIndex below. So just add aria-hidden and remove these. Also remove the blank lines please